labs/saw/arithmetic-verification/Makefile

```diff
@@ -0,0 +1,20 @@
+BINARY_NAME=arithmetic_unit_tests
+CC=clang
+
+
+all: add_tests
```

Comment:
Any reason *not* to include $(BINARY_NAME) in the `all` dependencies?

Reply:
I put it back in -- I kind of like having the binary show up when I build it,

Reply:
Don't you want `make all` to build *both* `$(BINARY_NAME)` and `add_tests`?  That's what I get from the README.md.